applications/MeshingApplication/custom_utilities/mmg_utility.h

```diff
@@ -1050,6 +1058,11 @@ class MmgUtility
             }
             else
             {
+                if (mFramework == Lagrangian)
```

Comment:
The displacement is already interpolated with the others variables. I assume that the coordinates are already computed, but I will interpolate them as well.